packages/build-info/src/build-systems/build-system.ts

```diff
@@ -33,6 +26,8 @@ export abstract class BaseBuildTool {
   name: string
   version?: string
   configFiles: string[] = []
+  /** If the command should be execute from the repository root */
```

Comment:
```suggestion
  /** If the command should be executed from the repository root */
```

Reply:
Addressed in https://github.com/netlify/build/pull/4977/commits/ac28711e57721593b35986c1b39480354cba73af